templates/harbor.yml.epp

```diff
@@ -112,6 +113,11 @@ jobservice:
   # Maximum number of job workers in job service  
   max_job_workers: <%= $max_job_workers %>
 
+<% if ersioncmp($cfg_version, '1.9.0') >= 0 { -%>
```

Comment:
missing v?

Reply:
is $cfg_version not the same as the on in config.pp